### PR TITLE
Update to openlayers 4.1.1, move legacy to ol2 directory

### PR DESCRIPTION
**Pull request by bartvde**
as per request https://github.com/openlayers/openlayers/issues/6860

### CODE REVIEW

**Comment by bartvde** [comment]
one issue still left that I need to fix in this PR, when you click docs there are assets missing
selection_279

**Comment by bartvde** [comment]
the assets are not in the ZIP archive, since the docs page already uses CDN, we might as well take these from the web instead with a sed replace I guess

**Comment by bartvde** [comment]
PR should now be ready for review

**Comment by kalxas** [comment]
Thank you @bartvde 
Since I do not see your name in the contributors list (https://github.com/OSGeo/OSGeoLive-doc/blob/master/contributors.csv) could you please send the following e-mail to the live-demo mailing list as per https://wiki.osgeo.org/wiki/Live_GIS_Disc#Git_migration ?

"Hi all, I'd like to contribute XXX to OSGeo-Live. I confirm that my contributions to OSGeo-Live will be compatible with the OSGeo-Live license guidelines at the time of contribution."

File: app-conf/mapnik/local.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
         unzip OpenLayers-2.9.zip
 -->
 
-<script src="http://localhost/openlayers/OpenLayers.js"></script>
+<script src="http://localhost/ol2/OpenLayers.js"></script>
```
Comment (kalxas) [inline app-conf/mapnik/local.html]: good catch :)

File: bin/install_openlayers.sh [hunk 2]
```diff
@@ -71,52 +71,59 @@ rm -rf "OpenLayers-$OL2_VERSION"
...
 #
 echo "\nUnzipping archive..."
-unzip -qo $OL3_ARCHIVE_FULL
-unzip -qo $OL3_ARCHIVE_DIST
+unzip -qo $OPENLAYERS_ARCHIVE_FULL
+unzip -qo $OPENLAYERS_ARCHIVE_DIST
 echo "Unzipping done"
 
 #
+# Replace assets url and workshop url
```
Comment (kalxas) [inline bin/install_openlayers.sh]: When offline this will cause a similar error as before, so its ok.

**Comment by bartvde** [comment]
@kalxas I've added myself to the csv with: https://github.com/OSGeo/OSGeoLive-doc/pull/198 is that enough?

**Comment by kalxas** [comment]
We have a policy to ask for a confirmation that you agree with the license. If you do not want to register to the mailing list, just post here so I can merge your pull requests. Sorry for the inconvenience.

**Comment by bartvde** [comment]
Hi all, I'd like to contribute openlayers to OSGeo-Live. I confirm that my contributions to OSGeo-Live will be compatible with the OSGeo-Live license guidelines at the time of contribution.

Signed @bartvde

**Comment by bartvde** [comment]
@kalxas see above, thanks

**Pull request merged**